pom.client.xml

```diff
@@ -428,6 +428,10 @@
               *.implementation*:
               com.azure.tools.checkstyle*
             </excludePackageNames>
+            <!-- CosmosSkip - This is temporary, BridgeInternal gets converted to implementation -->
```

Comment:
Right now, the goal is to get Comos v4 into master and successfully building in the pipelines which will unblock the development. There are still source changes that need to get made per the arch board review that will not be part of this PR. @kushagraThapar and I agreed that tagging things in the pom files, like plugin overrides or things that are temporary, with "CosmosSkip" so they can be easily identified. It is my understanding that everything tagged with CosmosSkip is going to be attached to a work item which is to be completed before GA.